src/android/BuiltinUserCache.java

```diff
@@ -380,9 +381,74 @@ public JSONObject getLocalStorage(String key, boolean withMetadata) throws JSONE
         }
     }
 
+    /**
+     * Gets all data in DB given a key and returns it in a JSONArray
+     * @param key
+     * @param withMetadata
+     * @return JSONArray of data from SQL database
+     * @throws JSONException
+     */
+    public JSONArray getLocalStorageArr(String key, boolean withMetadata) throws JSONException {
```

Comment:
nit: remove unused function

Reply:
I still use this function in my code, specifically to see how many entries are saved under a `KEY`.

```
JSONArray values = getLocalStorageArr(key, false);
            if (values.length() >= BOUND){
```
We do have `getLocalStorage` but it only returns the first element of the array. Let me know if there is a different function you may want me to use instead.